spec/App/MarkdownSpec.php

```diff
@@ -21,11 +21,34 @@
 
 use PhpSpec\ObjectBehavior;
 use Prophecy\Argument;
+use App\Writer;
+use App\Reader;
 
 class MarkdownSpec extends ObjectBehavior
 {
-    function it_is_initializable()
```

Comment:
`it_is_initializable()` with `$this->shouldHaveType('App\Markdown');` content still ok here, no need to remove the method.